pcdet/datasets/argo2/argo2_utils/so3.py

```diff
@@ -6,7 +6,7 @@
 from math import pi as PI
 
 
-@torch.jit.script
+# @torch.jit.script
```

Comment:
note: this JIT compilation did _not_ work for me inside the CUDA 11.6 docker image; I'm not sure exactly why.

It looks like this file has a lot of similary-commented-out `@torch.jit.script` calls, I'm guessing that it's a not-fully-implemented option.